build/doc.go

```diff
@@ -2,5 +2,5 @@ package build
 
 import (
 	// Import alpha-build-machinery to get standard makefile targets
-	_ "github.com/openshift/library-go/alpha-build-machinery"
+	_ "github.com/openshift/build-machinery-go"
```

Comment:
for me, this is OK to go into the bump commit. iirc `go mod tidy` will clean up the `go.mod` file if it doesn't find a direct or indirect reference.

Reply:
You could run `go mod tidy` and only split the files into correct commits after, or do the reverse order.

We are not enforcing this anywhere else yet, but there are strict requirements in origin to touch only glide files and vendor.

I think the best reason for splitting files correctly into bump commits is review. There is CI check named `verify deps` that makes sure `go mod tidy` was called and that all the files in vendor/ match what's written in `go.{mod,sum}`. With that the only files that need to be reviewed are `go.{mod,sum}`.